Refactor/about page with dep updates

uzomaolive opened the pull request:
<!-- Love osca? Please consider supporting our collective:
👉  https://opencollective.com/osca/donate -->

#### What does this PR do

This PR refactors the layout and styling of the `/about-us` page **to improve visual alignment**, updates key dependencies including `classnames`, performs related code cleanup within the component, and includes an update to the `sitemap.xml` timestamps.

#### Description of Task to be completed

  * **Refactor About Us Page:**
      * Restructured the component's JSX for better readability and flow.
      * Removed the complex layout relying on negative margins (`lg:-mt-40`, `lg:-mb-40`) for section overlaps.
      * Relocated and restyled the "Who we are" section to be a distinct, centered content block below the hero image.
      * **Adjusted layout elements for better visual alignment and consistency across the page.**
      * Adjusted vertical spacing between content sections.
  * **Dependency Updates:**
      * Updated `classnames` package from `^2.3.1` to `^2.5.1`.
      * Updated `yarn.lock` file, adding integrity hashes and resolving dependency versions.
  * **Code Cleanup:**
      * Removed obsolete commented-out code blocks within the `AboutUsPage` component that related to the previous layout.
  * **Sitemap Update:**
      * Refreshed the `<lastmod>` timestamps in `sitemap.xml` to the latest generation date (`2025-04-24T...`).

#### How should this be manually tested

1.  Checkout this branch: `git checkout <your-branch-name>`
2.  Install dependencies: `yarn install` (or `npm install`)
3.  Start the development server: `yarn dev` (or `npm run dev`)
4.  Navigate to the `/about-us` page (`http://localhost:3000/about-us` or similar).
5.  **Verify Layout & Alignment:**
      * Confirm the "Who we are" section appears below the main banner image, is centered, and does not overlap the banner.
      * Check the "Mission" and "Our Vision" sections – ensure the text blocks align better with their corresponding images and adjacent content. Pay attention to horizontal and vertical alignment on larger screens.
      * Resize the browser window or use developer tools to check responsiveness on desktop, tablet, and mobile views. Ensure the layout adapts gracefully and elements remain well-aligned.
6.  **Verify Functionality:** Click any links on the page (e.g., "Check out our community") to ensure they still work.
7.  **(Optional) Verify Build:** Run `yarn build` (or `npm run build`) and ensure the process completes without errors related to the updated dependencies or code changes.

#### What are the relevant issues this PR belongs to
N/A

#### Any background context you want to add

The previous layout of the About Us page used several negative margins to create overlapping effects, which could be complex to maintain and sometimes led to inconsistent alignment. This refactoring simplifies the layout approach, aiming for better maintainability **and improved visual alignment** of page elements. Dependency updates were included as part of routine site maintenance. The sitemap update is an artifact of the build/generation process used during development or testing of these changes.

#### Screenshots (Optional)
##Before
![screencapture-oscafrica-org-about-us-2025-04-24-12_26_03 (1)](https://github.com/user-attachments/assets/3778203a-1685-45be-ba4a-c628bf3a248c)

##After
![screencapture-localhost-3001-about-us-2025-04-25-18_52_29 (1)](https://github.com/user-attachments/assets/3d39fa0c-3492-49df-9c9f-ef791924626c)

Figma Mockup 
https://www.figma.com/design/UHDB0tPaIugeV4pc8dT0Hs/osca-about-page?node-id=0-1&t=TGT45MqkMV8lzTxR-1

uzomaolive committed:
refactor(page/about): Detailed layout restructuring and styling overhaul for About Us page

This commit implements a comprehensive refactoring of the About Us page component, involving significant changes to its JSX structure, CSS class application (via Tailwind), and consequently, its visual presentation across different screen sizes. The primary goal is simplifying complex overlapping layouts and creating clearer content sections.

**Detailed Breakdown of Changes:**

1.  **"Who we are" Section Transformation:**
    * **Structural Relocation:** Removed the entire `div` containing the "Who we are" content from its original position within the first `<article>` tag (where it previously followed an `ImageBackground` component).
    * **New Placement:** Re-introduced the "Who we are" content within a new `div` structure placed directly under the main `<section className="w-full">` tag, making it a distinct top-level section rather than part of the first article.
    * **Layout Overhaul:**
        * Removed the previous layout classes: `relative`, `lg:w-3/5`, `lg:-mt-40`, `z-30`. This eliminates the side-aligned (3/5 width on large screens) appearance and the visual overlap with the preceding image caused by the negative top margin.
        * Introduced new layout structure: A wrapping `div.bg-white.py-20.px-6.lg:px-20.w-full` containing an inner `div.max-w-5xl.mx-auto.text-center`. This establishes a full-width background section with centered content constrained to `max-w-5xl`.
    * **Styling & Content Presentation:**
        * Applied `text-center` to the inner container, centering the `SectionTitle` and the paragraph text.
        * Adjusted padding: `py-36` changed to `py-20`; `px-10` changed to responsive `px-6 lg:px-20`.
        * Modified paragraph styling using Tailwind utilities: Added `text-sm md:text-base leading-relaxed` for responsive font sizing and line height.
    * **Associated Image:** The `ImageBackground` component within the first `<article>` now stands alone without the accompanying text block it was previously paired with visually via the negative margin.

2.  **"Mission" Section Text Block Adjustment:**
    * **Class Removal:** Specifically removed the `lg:-mb-40` class from the `div` containing the "Mission" title and paragraph.
    * **Visual Impact:** This eliminates the negative bottom margin on large screens. The text block will no longer visually extend downwards significantly past its natural content height or the height of the adjacent `ImageBackground` (defined by `lg:w-3/5`), preventing potential overlap with subsequent elements and altering the vertical rhythm.

3.  **"Our Vision" Section Text Block Adjustment:**
    * **Class Removal:** Similarly, removed the `lg:-mb-40` class from the `div` containing the "Our Vision" title and paragraph.
    * **Visual Impact:** Same effect as the "Mission" section adjustment – removes the artificial downward extension of the text block on large screens, normalizing its bottom boundary.

4.  **Inter-Section Spacing Refinement:**
    * **Margin Updates (Large Screens):**
        * The top margin for the `<article>` containing the "Our Vision" section was changed from `lg:mt-64` to `lg:mt-56`.
        * The top margin for the final `<article>` containing the "Our Community" section was changed from `lg:mt-64` to `lg:mt-60`.
    * **Effect:** Subtly reduces the vertical whitespace between these major sections on large viewports.

5.  **Main Content Section Wrapper Modification:**
    * **Class Removal:** Removed the `max-w-1440` class from the opening `<section>` tag that wraps the primary content articles (excluding the hero section).
    * **Layout Implication:** The section itself no longer enforces a maximum width of 1440px. Width control is now potentially delegated further up to the `Layout` component or handled more granularly by the individual content blocks within the section (e.g., the new `max-w-5xl` in "Who we are", the `lg:w-*` classes on image/text pairs).

In summary, this refactoring moves away from using negative margins for complex overlaps towards more distinct, flow-based content sections, particularly for the introductory "Who we are" text. It also adjusts vertical spacing and potentially alters the main content area's width constraints.

uzomaolive committed:
chore(page/about): remove commented-out code

Removes commented-out JSX blocks from the About Us page component. This commented code represented the previous layout structure before the recent refactoring and is no longer needed.

This cleanup improves code readability and removes dead code.

uzomaolive committed:
chore(page/about): remove commented-out code line

Remove a single commented-out line containing a previous version of the 'Mission' section's container div definition.

uzomaolive committed:
Updates the `classnames` package from `^2.3.1` to `^2.5.1`. This brings in the latest non-breaking changes, potential bug fixes, and improvements from versions 2.3.2 through 2.5.1.

uzomaolive committed:
chore(sitemap): update lastmod timestamps in sitemap.xml

Updates the `<lastmod>` timestamps for all URLs listed in the `sitemap.xml` file. The timestamps have been refreshed to `2025-04-24T21:04:35Z` (approximately).

uzomaolive committed:
style(about-us): remove trailing spaces to comply with linting rules

- Removed unnecessary trailing spaces in the `about-us.tsx` file.
- Ensures compliance with the `no-trailing-spaces` ESLint rule.
- Update sitemap.xml

uzomaolive commented:
@Princesso  @alabobriggs and @BolajiAyodeji I’ve created a PR to update the layout of the site. I’d really appreciate it if someone could take a moment to review it. Thanks!

alabobriggs commented on package.json:
```diff
@@ -30,7 +30,7 @@
     "@babel/core": "^7.14.3",
     "@types/styled-components": "^5.1.9",
     "autoprefixer": "^10",
-    "classnames": "^2.3.1",
```
revert this back to 2.3.1, that's a major update and it is not clear if it introduces any breaking changes.

alabobriggs commented on src/pages/about-us.tsx:
```diff
@@ -49,25 +49,30 @@ const AboutUsPage = () => {
         </StyledMainSection>
 
-        <section className="w-full max-w-1440 ">
+        <section className="w-full">
```
The `max-w-1440` is required so it doesn't span the entire screen on monitor displays and larger screen

alabobriggs reviewed (changes requested):
Looks good, left some minor comments

uzomaolive commented:
Thank you so much for reviewing this. I really appreciate your feedback and will make the necessary changes promptly.

uzomaolive committed:
  fix(ui): refine layout on About Us page
 
   - Adds `max-w-1440` to constrain section width.

    - Reduces large screen top margin on an article element (`lg:mt-40`).

    - Increases base font size of 'Who we are' paragraph to `text-base` (16px) for better mobile readability on the About Us page.

    - Includes minor code style cleanup (removed trailing space in className).

uzomaolive committed:
fix: downgrade classnames dependency

- Downgrades the `classnames` dependency from v2.5.1 to v2.3.1.

uzomaolive committed:
Updates the auto-generated `sitemap.xml` with current timestamps.

uzomaolive commented:
@alabobriggs Addressed the issues and updated the code. Please let me know if this looks better. Thanks for reviewing!

alabobriggs reviewed (approved):
LGTM

uzomaolive commented:
Thanks a lot for reviewing this @alabobriggs. I truly appreciate your feedback and support.
